dcicutils/beanstalk_utils.py

```diff
@@ -13,7 +13,7 @@
 from datetime import datetime
 from . import ff_utils
 from botocore.exceptions import ClientError
-from .misc_utils import PRINT
+from .misc_utils import PRINT, deprecated
```

Comment:
I'd like to see this deprecation stuff renamed. Deprecation is advisory and doesn't change semantics. You are making these obsolete, which I endorse.